go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/operator-framework/operator-registry
 
-go 1.19
+go 1.20
```

Comment:
Any reason not to bump to 1.21?

Reply:
k8s 1.28 is still on 1.20, as @varshaprasad96 mentioned in slack (https://kubernetes.slack.com/archives/C0181L6JYQ2/p1695306483677709?thread_ts=1695306384.252939&cid=C0181L6JYQ2) but really I was just bringing api/reg up to sync with o-l-m, which is on 1.20.